docs/configuration.rst

```diff
@@ -421,6 +421,15 @@ The following environment variables for the tracer are supported:
        v1.16.2: added with default of False
        v1.19.0: default changed to True
 
+   DD_TRACE_METHODS:
+     type: String
+     default: ""
+     description: |
+        Specify methods to trace. For example: ``mod.submod[method1,method2];mod.submod.Class[method1]``.
+        Note that this setting is only compatible with ``ddtrace-run``.
+     version_added:
+       v2.1.0:
```

Comment:
we might need to change this now that we're potentially delaying 2.0 release

Reply:
yeah 😕

Reply:
I don't think the date of 2.0 release is a blocker as that release branch is created. This can be merged as is.